src/shapesdemo/ShapeSubscriber.cpp

```diff
@@ -63,7 +63,7 @@ void ShapeSubscriber::onNewDataMessage(Subscriber* sub)
     Shape shape;
     shape.m_type = this->m_shapeType;
     SampleInfo_t info;
-    while(sub->takeNextData((void*)&shape.m_shape,&info))
+    while(sub->readNextData((void*)&shape.m_shape,&info))
```

Comment:
Why was this changed? Maybe you had this change locally and committed it by accident or is there a reason to change it?